src/services/database.ts

```diff
@@ -127,3 +129,34 @@ export const postTechRecord = async (request: TechrecordGet): Promise <Techrecor
     throw new Error('database client failed getting data');
   }
 };
+export const archiveOldCreateCurrentRecord = async (recordToArchive: TechrecordGet, recordToCreate: TechrecordGet): Promise<string | Error> => {
+  logger.info('Preparing Transact Items');
+
+  const transactParams: TransactWriteItemsInput = {
+    TransactItems: [
+      {
+        Put: {
+          TableName: tableName,
+          Item: marshall(recordToArchive),
+          ConditionExpression: 'attribute_exists(systemNumber) AND attribute_exists(createdTimestamp)',
+        },
+      },
+      {
+        Put: {
+          TableName: tableName,
+          Item: marshall(recordToCreate),
+        },
+      },
+    ],
+  };
+
+  try {
+    await ddbClient.send(new TransactWriteItemsCommand(transactParams));
+
+    return 'records updated';
```

Comment:
Is this return actually used for anything? If not should it not just ```return```

Reply:
I've set it to return undefined. as the linter doesn't like it when I just return